libraries/ESP8266HTTPClient/examples/PostHttpsClient/PostHttpsClient.ino

```diff
@@ -0,0 +1,86 @@
+/**
+   PostHttpsClient.ino
+    Created on: 28.06.2021
+*/
+
+#include <Arduino.h>
+#include <ESP8266WiFi.h>
+#include <ESP8266WiFiMulti.h>
+#include <WiFiClientSecureBearSSL.h>
+#include <ESP8266HTTPClient.h>
+
+// Fingerprint for demo URL, expires on September 5, 2021, needs to be updated well before this date
+const uint8_t fingerprint[20] = {0xbb, 0x88, 0x7f, 0x7c, 0x77, 0xc2, 0x59, 0x97, 0xb7, 0x00, 0x35, 0x74, 0x50, 0x47, 0x7e, 0x67, 0x42, 0x02, 0x2f, 0xf0};
+const char* URL = "https://www.reseau-astuce.fr/fr/horaires-a-larret/28/StopTimeTable/NextDeparture";
+const char* REQUEST = "destinations=%7B%221%22%3A%22Technop%C3%B4le+SAINT-ETIENNE-DU-ROUVRAY%22%7D&stopId=102154&lineId=175&sens=1";
+
+const char* WIFI_SSID = "myWifiSSID";
+const char* WIFI_PWD = "myWifiPassword";
+
+ESP8266WiFiMulti WiFiMulti;
+
+
+
+void setup() {
+
+  Serial.begin(115200);
+  Serial.println();
+  Serial.println();
+  Serial.println();
+
+  WiFi.mode(WIFI_STA);
+  WiFiMulti.addAP(WIFI_SSID, WIFI_PWD);
+
+  Serial.println("[WIFI] Connecting to WiFi ...");
+  while (WiFiMulti.run() != WL_CONNECTED) {
+    Serial.print('.');
+    delay(1000);
+  }
+
+  Serial.println();
+  Serial.print("[WIFI] Connected with IP : ");
+  Serial.println(WiFi.localIP());
+  WiFi.setAutoReconnect(true);
+  WiFi.persistent(true);
+
+}
+
+
+
+void loop() {
+
+  if ((WiFiMulti.run() == WL_CONNECTED)) {
+    Serial.println("[HTTPS] begin...");
+
+    std::unique_ptr<BearSSL::WiFiClientSecure>client(new BearSSL::WiFiClientSecure);
+    client->setInsecure();                  // Ignore SSL certificate
+    //client->setFingerprint(fingerprint);  //Use SSL
```

Comment:
It is prefered to use secure connections in examples. 
You may leave the `setInsecure()` commented though.